java/rocksjni/options.cc

```diff
@@ -2302,6 +2302,35 @@ jlong Java_org_rocksdb_Options_bgerrorResumeRetryInterval(JNIEnv*, jclass,
   return static_cast<jlong>(opt->bgerror_resume_retry_interval);
 }
 
+/*
+ * Class:     org_rocksdb_Options
+ * Method:    setDailyOffpeakTimeUTC
+ * Signature: (JLjava/lang/String;)V
+ */
+void Java_org_rocksdb_Options_setDailyOffpeakTimeUTC(JNIEnv* env, jclass,
+                                                     jlong jhandle,
+                                                     jstring jtimeutc) {
+  auto* opt = reinterpret_cast<ROCKSDB_NAMESPACE::Options*>(jhandle);
+  const jsize jtimesz = env->GetStringUTFLength(jtimeutc);
```

Comment:
I thought we used to have utility method for converting strings.

Reply:
You are right. I've picked up that utility.